flask_login/utils.py

```diff
@@ -238,7 +238,7 @@ def login_required(func):
         @app.route('/post')
         @login_required
         def post():
-            pass
+            return a_response
```

Comment:
Where's `a_response` defined? Instead of returning an unseen variable just return a string?